### PR TITLE
Adds an experiment to run the InstrumentationVerifier on a project

### PR DESCRIPTION
**Pull request by vulder**

### CODE REVIEW

File: varats/varats/experiments/vara/feature_experiment.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
+"""Base class experiment and utilities for experiments that work with
+features."""
+# TODO: figure out where to put the file
```
Comment (vulder) [inline varats/varats/experiments/vara/feature_experiment.py]:
```suggestion
```

File: varats/varats/data/reports/instrumentation_verifier_report.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+"""Instrumentation verifier report implementation for checkif if projects get
+correctly instrumented."""
+
+from varats.report.report import BaseReport
+
+
+class InstrVerifierReport(BaseReport, shorthand="IVR", file_type="txt"):
+    """An instrumentation verifier report for testing how well project where
+    instrumented."""
```
Comment (jonas-kaufmann) [inline varats/varats/data/reports/instrumentation_verifier_report.py]:
```suggestion
    """An instrumentation verifier report for testing how well projects were
    instrumented."""
```

File: varats/varats/experiments/vara/feature_experiment.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        """
+        Returns the ldflags needed to instrument projects with VaRA during LTO.
+
+        Returns: ldflags for VaRA LTO support
+        """
+        return ["-flto"]
+
+
+class RunVaRATracedWorkloads(ProjectStep):  # type: ignore
+    """Executes the traced project binaries over the specified workloads."""
```
Comment (jonas-kaufmann) [inline varats/varats/experiments/vara/feature_experiment.py]:
```suggestion
    """Executes the traced project binaries on the specified workloads."""
```

**Review by jonas-kaufmann — APPROVED** [review]
Two small typos otherwise lgtm.

**Review by boehmseb — APPROVED** [review]
lgtm

**Review by Aufrichtig98 — COMMENTED** [review]
gtm

**Pull request merged**